test/Window/Event.test.cpp

```diff
@@ -50,14 +55,22 @@ TEST_CASE("[Window] sf::Event")
 
     SECTION("Construction")
     {
-        SECTION("Template constructor")
+        SECTION("Template constructor resize")
         {
             const sf::Event event = sf::Event::Resized{{1, 2}};
             CHECK(event.is<sf::Event::Resized>());
             CHECK(event.getIf<sf::Event::Resized>());
             const auto& resized = *event.getIf<sf::Event::Resized>();
             CHECK(resized.size == sf::Vector2u(1, 2));
         }
+        SECTION("Template constructor window move")
+        {
+            const sf::Event event = sf::Event::WindowMoved{{1, 2}};
+            CHECK(event.is<sf::Event::WindowMoved>());
+            CHECK(event.getIf<sf::Event::WindowMoved>());
+            const auto& windowMoved = *event.getIf<sf::Event::WindowMoved>();
+            CHECK(windowMoved.position == sf::Vector2i(1, 2));
+        }
```

Comment:
```suggestion
```

Reply:
Chris is suggesting to delete those lines.